test/unit/opensn_unit_test.h

```diff
@@ -0,0 +1,10 @@
+#pragma once
+
+#include <gtest/gtest.h>
+
+class OpenSnUnitTest : public ::testing::Test
```

Comment:
@andrsd should this be `opensn` namespace-d?

Reply:
I don't think there is a reason for that, since the unit testing binary is not being deployed